src/clr-addons/history/history.provider.ts

```diff
@@ -0,0 +1,8 @@
+import { InjectionToken } from '@angular/core';
+import { ClrHistoryModel } from './history-model.interface';
+
+export const HISTORY_PROVIDER = new InjectionToken<any>('HISTORY_PROVIDER');
+
+export abstract class HistoryProvider {
+  abstract modifyHistoryEntries(entries: ClrHistoryModel[]): void;
```

Comment:
pls leave a comment what this method is supposed to do. as I'm not sure what I have to expect from it :)
and I'm a big fan of return values if anything is changed, instead of modifying parameters

Reply:
Done